Dockerfile

```diff
@@ -1,12 +1,34 @@
-FROM quay.io/hotosm/base-python-image as base
-LABEL version=0.1
-LABEL maintainer="HOT Sysadmin <sysadmin@hotosm.org>"
-LABEL description="Builds backend docker image"
+ARG ALPINE_IMG_TAG=3.17
+ARG PYTHON_IMG_TAG=3.9
 
-WORKDIR /usr/src/app
+FROM docker.io/python:${PYTHON_IMG_TAG}-alpine${ALPINE_IMG_TAG} as base
```

Comment:
@spwoodcock Just for context, one of the reasons we switched from Docker to Quay was that Docker was rate limiting us. Our pipelines build docker images every push (probably a bad idea and will be fixed) and has been pulling from docker.io very often - hitting the rate limits. Just FYI, no action needed.

Reply:
Yeah, that's a headache. I'll find a fix.